src/Peachpie.CodeAnalysis/Symbols/CoreType.cs

```diff
@@ -179,6 +179,7 @@ public CoreTypes(PhpCompilation compilation)
             IPhpArray = Create("IPhpArray");
             IPhpEnumerable = Create("IPhpEnumerable");
             IPhpCallable = Create("IPhpCallable");
+            PhpCallback = Create("PhpCallback");
```

Comment:
can be removed if you move the helper method into `Operators`

Reply:
OK, moved in 8615dc96.